configs/model/cfa.yaml

```diff
@@ -0,0 +1,107 @@
+# This is the config file for the new Anomalib CLI.
+# To use this, run the following command:
+# anomalib fit --config ./config/model/stfpm.yaml
+ckpt_path: null
+seed_everything: null
+
+data:
+  class_path: anomalib.data.MVTec
+  init_args:
+    root: ./datasets/MVTec
+    category: bottle
+    image_size: [224, 224]
+    train_batch_size: 4
+    test_batch_size: 4
+    num_workers: 8
+    task: segmentation
+    transform_config_train: null
+    transform_config_val: null
+    seed: 0
+    create_validation_set: false
+
+model:
+  class_path: anomalib.models.Cfa
+  init_args:
+    input_size: [224, 224]
+    backbone: wide_resnet50_2
+    gamma_c: 1
+    gamma_d: 1
+
+optimizer:
+  class_path: torch.optim.AdamW
+  init_args:
+    lr: 1e-3
+    weight_decay: 5e-4
+    amsgrad: true
+
+metrics:
+  adaptive_threshold: true
+  default_image_threshold: null
+  default_pixel_threshold: null
+  image_metric_names:
+    - AUROC
+  pixel_metric_names:
+    - AUROC
+    - AUPRO
+  normalization_method: min_max
+
+visualization:
+  show_images: False # show images on the screen
+  save_images: False # save images to the file system
+  log_images: False # log images to the available loggers (if any)
+  mode: full # options: ["full", "simple"]
+
+trainer:
+  callbacks:
+    - class_path: pytorch_lightning.callbacks.EarlyStopping
+      init_args:
+        patience: 5
+        monitor: pixel_AUROC
+        mode: max
+  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
+  accumulate_grad_batches: 1
+  amp_backend: native
+  auto_lr_find: false
+  auto_scale_batch_size: false
+  auto_select_gpus: false
+  benchmark: false
+  check_val_every_n_epoch: 1
+  default_root_dir: null
+  detect_anomaly: false
+  deterministic: false
+  devices: 1
+  enable_checkpointing: true
+  enable_model_summary: true
+  enable_progress_bar: true
+  fast_dev_run: false
+  gpus: null # Set automatically
+  gradient_clip_val: 0
+  ipus: null
+  limit_predict_batches: 1.0
+  limit_test_batches: 1.0
+  limit_train_batches: 1.0
+  limit_val_batches: 1.0
+  log_every_n_steps: 50
+  log_gpu_memory: null
+  max_epochs: 30
```

Comment:
Is this what the paper recommends? We have `max_epochs` as 100 for all the models except for those where the paper requires more.

Reply:
Same as above.